src/main/resources/org/jenkinsci/plugins/github/admin/GitHubHookRegisterProblemMonitor/message.groovy

```diff
@@ -1,11 +1,24 @@
 package org.jenkinsci.plugins.github.admin.GitHubHookRegisterProblemMonitor
 
+import hudson.util.VersionNumber
+import jenkins.model.Jenkins
+
 def f = namespace(lib.FormTagLib)
 
-div(class: 'alert alert-warning') {
-    form(method: 'post', action: "${rootURL}/${my?.url}/act", name: my?.id) {
-        f.submit(name: 'yes', value: _('view'))
-        f.submit(name: 'no', value: _('dismiss'))
+if (Jenkins.getVersion().isNewerThan(new VersionNumber("2.88"))) {
```

Comment:
`2.88` is only a tentative version, I'm waiting for getting this https://github.com/jenkinsci/jenkins/pull/2857 already merged

Reply:
Why do you need to wait that?
`alert` and `alert-warning` is available in Jenkins 1.633 (https://github.com/jenkinsci/ssh-slaves-plugin/pull/70#pullrequestreview-63935217)

Reply:
what if it will be lts? Is side effects worse it?

Reply:
@lanwen I don't think so. With this approach if the plugin is running on Jenkins version `< 2.88` we see the current appearance, but if the plugin is running on Jenkins `> 2.88` we see the new design.